src/metabase/search/fulltext.clj

```diff
@@ -13,7 +14,7 @@
 (defmethod supported-db? :default [_] false)
 
 (defmethod supported-db? :postgres [_]
-  (public-settings/experimental-fulltext-search-enabled))
+  (or (public-settings/experimental-fulltext-search-enabled) config/is-test?))
```

Comment:
This might break more things in CI, let's see.

Reply:
why do we want this?

Reply:
So that the new search tests run in CI

Reply:
oh it's false by default?

Reply:
how about making the `experimental-fulltext-search-enabled` setting default to true in test instead?

Reply:
It does, so I'm going to update the setting default in another PR.

It seems like there is order coupling between the tests behind at least some of the failures.